.changeset/smooth-dogs-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-table": minor
```

Comment:
I'd say this is a `patch` since no new feature was added.

Reply:
Thanks @gyfchong, that's done.